src/server/api.md

```diff
@@ -4,9 +4,8 @@
 
 ```ts
 
-/// <reference types="node" />

```

Comment:
Nice! Something is right if `/// <reference types="node" />` is removed. Love it.